apps/builder/app/builder/features/settings-panel/props-section/use-props-logic.ts

```diff
@@ -218,14 +218,17 @@ export const usePropsLogic = ({
     return { prop: saved, propName: name, meta };
   });
 
-  const initialProps: PropAndMeta[] = Array.from(initialPropsNames, (name) => {
+  const initialProps: PropAndMeta[] = [];
+  for (const name of initialPropsNames) {
     const saved = getAndDelete<Prop>(unprocessedSaved, name);
     const known = getAndDelete(unprocessedKnown, name);
 
     if (known === undefined) {
-      throw new Error(
+      // eslint-disable-next-line no-console
+      console.error(
```

Comment:
I wish we would throw locally but log in prod, otherwise risk is still there we won't see the log during dev